scripts/ci/install-rust.sh

```diff
@@ -2,8 +2,17 @@
 
 set -ex
 
+git clone https://github.com/influxdata/flux.git flux-repo
```

Comment:
Instead of the full git-clone, could we `wget` the one config file we need?

Reply:
I guess git gives us a bunch of integrity checks...

Reply:
we could but then we would be pulling the very latest version. I'd prefer to only get the version from the latest tag, so we aren't following flux's cutting edge, but I could be persuaded otherwise.

Reply:
maybe i'm making a bad argument if we are always building with the latest commit from flux

Reply:
Yeah your point about assurances from git is a good one too